addons/l10n_in_edi/utils.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import requests
+import lxml
+
+def verify_signed_invoice(json_data):
+    url = "https://einvoice1.gst.gov.in/Others/VSignedInvoice"
+    get_res = requests.get(url)
+    cookies = get_res.cookies.get_dict()
+    html_text = get_res.text
+    html = lxml.html.fromstring(html_text)
+    try:
+        requestverificationtoken = html.xpath('//input[@name="__RequestVerificationToken"]/@value')[0]
+    except:
+        return html_text
+    payload = {
+        'submit': 'Upload',
+        '__RequestVerificationToken': requestverificationtoken,
+    }
```

Comment:
```suggestion
    requestverificationtoken = html.xpath('//input[@name="__RequestVerificationToken"]/@value')
    if not requestverificationtoken:
        return html_text
    payload = {
        'submit': 'Upload',
        '__RequestVerificationToken': requestverificationtoken[0],
    }
```
If the try except is there only for the case where ``html.xpath(...)`` would return an empty list.

Reply:
done